dvc/api.py

```diff
@@ -96,33 +105,67 @@ def _make_repo(repo_url, rev=None):
 
 
 def summon(name, repo=None, rev=None, summon_file="dvcsummon.yaml", args=None):
+    """Instantiate an object described in the summon file."""
+    with prepare_summon(
+        name, repo=repo, rev=rev, summon_file=summon_file
+    ) as desc:
+        try:
+            summon = SUMMON_PYTHON_SCHEMA(desc.obj["summon"])
```

Comment:
deepsource is complaining with naming a variable the same way as the method.

What about renaming it for `info`

Reply:
Renamed to `summon_dict`.